test/arrays/ferguson/tuples/return-tuple-slices.future

```diff
@@ -0,0 +1,4 @@
+bug: returning tuple of slices results in compilation error
+
+It should return a tuple of copies of the slices.
```

Comment:
Pre-existing condition. The problem is that computeNonRefTuple isn't taking in to account that autoCopy for slices returns a different type. I was just thinking about the related code and figured we probably had an error here...